luaui/RmlWidgets/gui_awards/gui_awards.rcss

```diff
@@ -0,0 +1,171 @@
+* {
+    font-family: "Exo 2";
+    font-size: 18px;
+    color: white;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+  background-color: rgba(255,255,255,0.7)
+}
+
+table {
+  box-sizing: border-box;
+  display: table;
+}
+
+tr {
+  box-sizing: border-box;
+  display: table-row;
+}
+
+td {
+  box-sizing: border-box;
+  display: table-cell;
+  text-align: center;
+  vertical-align: top;
+}
+
+thead, tbody, tfoot {
+  display: table-row-group;
+}
+
+body, div, p, hr, pre,
+h1, h2, h3, h4, h5, h6,
+tabset tabs {
+    display: block;
+}
+
+h1, h2, h3 {
+  font-weight: bold;
+}
+
+img {
+  height: 100px;
+
+}
+
+span {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
+
+p, span {
+  vertical-align: center;
+}
+
+td.align-left {
+  text-align: left;
+  vertical-align: middle;
+}
+
+#main-container {
```

Comment:
avoid % values as much as possible. Remember we want to limit the amount of calculations that have to happen each time this gets laid out. Instead to centre your widget use

```
position: absolute;
margin: 0 auto;
width: 750dp;
```